zoleotracker/databaseSQL.py

```diff
@@ -0,0 +1,57 @@
+import mysql.connector
+from mysql.connector import Error
+
+def create_db_connection(host_name, user_name, user_password, db_name):
+    connection = None
+    try:
+        connection = mysql.connector.connect(
+            host=host_name,
+            user=user_name,
+            passwd=user_password,
+            database=db_name
+        )
+        print("MySQL Database connection successful")
+    except Error as err:
+        print(f"Error: '{err}'")
+
+    return connection
+
+def check_for_table(connection, table_name):
+    cursor = connection.cursor()
+    try:
+        cursor.execute("SHOW TABLES LIKE '%s' ", table_name)
```

Comment:
this allows for sql injection. Anytime you give someone the permission to put a string into sql, you have this vulnerability. several mitigating strategies exists, I'd create an issue to come back and fix this. 

Priority order:
1. Get stuff working while you're learning
2. Secure

Reply:
for example if I called `check_for_table(conn, "foo'; DROP ALL TABLES;")`